mods/tuxemon/maps/taba_town.tmx

```diff
@@ -670,7 +670,7 @@
     <property name="cond2" value="is char_facing player,down"/>
     <property name="cond3" value="not variable_set moveon:yes"/>
     <property name="cond4" value="not variable_set timetomove:finite"/>
-    <property name="cond5" value="is has_item player,potion, equals,2"/>
+    <property name="cond5" value="is has_item player,potion"/>
```

Comment:
You shouldn't have any credits to purchase items yet at this point in time, but regardless, `greater_or_equal` would probably be a better fit since as it stands currently, a single potion would fulfill the requirements: https://wiki.tuxemon.org/Event_Reference#has_item

Reply:
I could def. buy more potions, this happened to be me in a playthrough.

You're right. I just wasn't very familiar with the code. I'll update.